docs/directory_structures.md

```diff
@@ -0,0 +1,76 @@
+# Directory structures for Lightning Pose
+
+## Data directory structure
+
+Lightning Pose requires data to be stored in a particular manner.
+Also see the example dataset provided in `lightning-pose/data/mirror-mouse-example`.
+
+    /path/to/project/
+      ├── labeled-data/
+      ├── videos/
+      └── CollectedData.csv
+
```

Comment:
`<YOUR_LABELED_FRAMES>.csv` instead of collected data.

`<YOUR_LABELED_FRAMES>.csv`: a table with keypoint labels (rows: frames, columns: keypoint names (and xoordinates)).